### PR TITLE
Begin PR for README updates

### PR DESCRIPTION
**Pull request by seanmooretechwriter**
Opening PR for updates to README file to address the Git task:

https://github.com/streamingfast/substreams/issues/84

### CODE REVIEW

**Comment by seanmooretechwriter** [comment]
Base README in place. Is this adequate or should we add more?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,60 @@
...
-This is a command line tool to quickly sync a substreams to a files.
+## Description
+
+`substreams-sink-files` is a tool that allows developers to pipe data extracted from a blockchain into various types of local or Cloud files-based persistence solutions.
+
+## Prerequisites
+
+- Go installation and compiler
+- Rust installation and compiler
+- Cloned `substreams-sink-files` repository
```
Comment (maoueh) [inline README.md]: Those three are not true, Go compiler is not required unless the person wants to compile from source. Rust is also not needed, somehow could run a pre-built `.spkg`. Let's remove those three and `Knowledge of blockchain development`, the other two stays.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,60 @@
...
+- Knowledge of blockchain development
+- Cloud-based file storage mechanism (optional)
+
+## Installation
+
+Install `substreams-sink-files` by using the pre-built binary release [available in the official GitHub repository](https://github.com/streamingfast/substreams-sink-files/releases).
+
+Extract `substreams-sink-files` into a folder available in your PATH.
+
+The binary file is installed in your GO_PATH, typically \$HOME/go/bin. Make sure this folder is included in your PATH environment variable.
```
Comment (maoueh) [inline README.md]: That is incorrect. It goes in `GOPATH` (and not `GO_PATH`) only when compiling/installing from source directly. When downloading from a pre-built binary, `GOPATH` as no impact at all. This sentence can be removed, I've proposed a small change for the other line.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,60 @@
...
+- Cloned `substreams-sink-files` repository
+- A Substreams module prepared for a files-sink
+- Knowledge of blockchain development
+- Cloud-based file storage mechanism (optional)
+
+## Installation
+
+Install `substreams-sink-files` by using the pre-built binary release [available in the official GitHub repository](https://github.com/streamingfast/substreams-sink-files/releases).
+
+Extract `substreams-sink-files` into a folder available in your PATH.
```
Comment (maoueh) [inline README.md]:
```suggestion
Extract `substreams-sink-files` into a folder and ensure this folder is reference globally via your `PATH` environment variable.
```

File: README.md [hunk 4]
```diff
@@ -1,3 +1,60 @@
...
+
+The binary file is installed in your GO_PATH, typically \$HOME/go/bin. Make sure this folder is included in your PATH environment variable.
+
+## Using the `substreams-sink-files` tool
```
Comment (maoueh) [inline README.md:18]: Let's update this section with latest content of tutorial, so we talk about output bundles, cursor and other nice thing you have there.

I'm fine also having quicker show up here with links back to the tutorial.

**Review by maoueh — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (seanmooretechwriter) [reply README.md]: Updated.

File: README.md (hunk 2 above)
Comment (seanmooretechwriter) [reply README.md]: Updated. This was pulled from the file sink KV content written by Steph.

File: README.md (hunk 3 above)
Comment (seanmooretechwriter) [reply README.md]: Updated.

File: README.md (hunk 4 above)
Comment (seanmooretechwriter) [reply README.md:18]: I ported the Cursors, Cloud-based storage and Limitations from the tutorial into the README. Is this adequate?

**Pull request merged**